faults.csv

```diff
@@ -1,63 +1,63 @@
+FBS,Piezo detune and cavity feedback not regulating,This alarms if the detune measurement used by the piezo feedback is < -50 Hz or > 50 Hz and the cavity amplitude or phase measurement is out of tolerance.,,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,FBSTATSUM,,2,2,Piezo detune and cavity feedback not regulating
+PZO,Piezo feedback problem Alarm,Piezo Feedback is indicating a problem,Run auto setup,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=3,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PZT:FBSTATSUM,,2,2,
+PZO,Piezo feedback problem Warning,Piezo Feedback is indicating a problem,Run auto setup,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=3,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PZT:FBSTATSUM,,1,1,Piezo feedback problem
+OVR,SSA Overrange,The SSA is unable to output enough power to attain the desired amplitude. Changing the reactive power fraction can help. ,"Run auto setup. If that fails, try increasing reactive power fraction to no more than 0.5 (will need to press Go). If that fails, CATER",CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=6,SELCHAR=0",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,ASETSUB.VALQ,,1,2,SSA Overrange
+USL,Upstream liquid level out of tolerance Alarm,Cryomodule liquid level out of tolerance,Call on shift cryo operator,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CLL:CM{CRYOMODULE}:2601:US:,LVL.SEVR,,2,2,
+USL,Upstream liquid level out of tolerance Warning,Cryomodule liquid level out of tolerance,Call on shift cryo operator,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CLL:CM{CRYOMODULE}:2601:US:,LVL.SEVR,,1,1,Upstream liquid level out of tolerance
+USH,Upstream HOM temperature is too high,Upstream HOM temperature is too high (could indicate an impending quench),Turn off RF,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CTE:CM{CRYOMODULE}:1{CAVITY}18:UH:,TEMP.SEVR,,2,2,Upstream HOM temperature is too high
+DSH,Downstream HOM temperature too high,Downstream HOM temperature too high (could indicate an impending quench),Turn off RF,CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CTE:CM{CRYOMODULE}:1{CAVITY}20:DH:,TEMP.SEVR,,2,2,Downstream HOM temperature too high
+AOT,Amplitude out of tolerance,The amplitude readback is too far away from the desired amplitude,Run auto setup,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,SELCHAR=3,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,AACTMEAN.SEVR,0,,2,Amplitude out of tolerance
+POT,Phase out of tolerance,The phase readback is too far away from the desired phase,Check that this isn't being caused by beam feedbacks,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PACTMEAN.SEVR,0,,2,Phase out of tolerance
+QBP,Quench Interlock Bypassed,Quench interlock bypassed,Unbypass the quench interlock,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,QUENCH_BYP_RBV,,1,1,Quench Interlock Bypassed

```

Comment:
I disagree. Unbypassing interlocks should not be done without expert instruction

Reply:
What would you suggest instead?

Reply:
I don't think that we will leave cavities in QBP without a good reason, or while actively doing something. So it becomes more of a 'Check with SRF person why this is bypassed or if it has been forgotten'

Reply:
I guess that assumes _we_ changed it. I was envisioning rogue physicist